frontend/lib/js/category-trees/selectors.js

```diff
@@ -0,0 +1,33 @@
+// @flow
+
+import { type TreeNodeIdType }   from "../common/types/backend";
+import { type SearchType }       from "./reducer";
```

Comment:
Being picky again... I'd vote for using single quotes here

Reply:
right.